components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -91,7 +91,9 @@ export function WorkspaceEntry({ desc, model, isAdmin, stopWorkspace }: Props) {
         </ItemFieldIcon>
         <ItemField className="w-3/12 flex flex-col">
             <a href={startUrl.toString()}><div className="font-medium text-gray-800 dark:text-gray-200 truncate hover:text-blue-600 dark:hover:text-blue-400">{ws.id}</div></a>
-            <a href={project ? 'https://' + project : undefined}><div className="text-sm overflow-ellipsis truncate text-gray-400 dark:text-gray-500 hover:text-blue-600 dark:hover:text-blue-400">{project || 'Unknown'}</div></a>
+            <Tooltip content={project ? 'https://' + project : ''}>
```

Comment:
**issue**: Could we enforce a max width for the tooltip component as the tooltip overflows the container width when the path is long?

<img width="1255" alt="Frame 38" src="https://user-images.githubusercontent.com/120486/130468250-b8d7d210-cdcb-4cc3-8737-145b48d536e0.png">

Reply:
Haha I wrote my comment just when you wrote this one.